sanitizer/search_fallback.js

```diff
@@ -1,29 +0,0 @@
-var sanitizeAll = require('../sanitizer/sanitizeAll'),
```

Comment:
yeah I agree with @orangejulius here. it seems like a big and unrelated change to make as part of the debugging work. would be best in a separate PR.